src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final int INDEX_OF_VALUE = 1;
+    private static final int INDEX_OF_WORD = 0;
+    private static final String BUY_WORD = "buy";
+    private static final String CSV_DELIMITER = ",";
+    private static final String RESULT_WORD = "result";
+    private static final String SUPPLY_WORD = "supply";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalBuy = 0;
+        int totalSupply = 0;
+
```

Comment:
delete all empty lines in the method. your code, due to good naming, should be understandable without additional separators

Reply:
fixed.